apps/wg-easy/config.json

```diff
@@ -11,7 +11,7 @@
   },
   "id": "wg-easy",
   "tipi_version": 5,
```

Comment:
Could you also bump tipi_version to 6? This will trigger the update to users

Reply:
sure, forgot about that